doc/man/maestro-config.1

```diff
@@ -12,6 +12,29 @@ maestro-config \- set up or alter your Maestro configuration files
 
 .SH DESCRIPTION
 
+.PP
+The
+.I maestro config
+command should be the first command run after installing Maestro. Any Maestro deployment depends on the account information in the hidden directory created by the initial invocation of
+.I maestro config.
+
+.PP
+.I maestro config
+checks for creation of a hidden
+.I .maestro
+directory created in the users home directory and also the existence of a file
+.I aws_account_info.json
+. If the directory and file exists, and there is information present, the values present are presented to the user when prompted for account information. Otherwise, a user will simply be prompted for appropriate account information (i.e. in the case this is the initial invocation of the command).
+
+.PP
+The pertinent pieces of account information that
+.I maestro config
+is concerned with are the AWS account number and region. The account number is a 12 digit number, and the user is required to input said number in the format
+.I XXXXXXXXXXXX
+
```

Comment:
There should probably be a period at the end of this sentence

Reply:
Added